pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -67,6 +68,10 @@ type Instance struct {
 	servers.Server
 }
 
+type ProviderNetwork struct {
```

Comment:
guess I missed some mail and I asked whether ProviderNetwork can be renamed because openstack provider network has special meaning...

Reply:
Didnt see that, but no problem. Easy fix.